test/test.config.js

```diff
@@ -15,85 +15,21 @@
  * limitations under the License.
  */
 
-const fs = require('fs');
 const path = require('path');
-const rm = require('rimraf').sync;
 const utils = require('./utils');
-const {TestServer} = require('../utils/testserver/');
-const {Environment} = require('../utils/testrunner/Test');
+const {DefaultBrowserOptionsEnvironment, ServerEnvironment, GoldenEnvironment, TraceTestEnvironment} = require('./environments.js');
 
 const playwrightPath = path.join(__dirname, '..');
 
-const serverEnvironment = new Environment('TestServer');
-serverEnvironment.beforeAll(async state => {
-  const assetsPath = path.join(__dirname, 'assets');
-  const cachedPath = path.join(__dirname, 'assets', 'cached');
+const extraLogger = utils.createTestLogger(valueFromEnv('DEBUGP', false), null, 'extra');
```

Comment:
This should be one logger per worker thread.

Reply:
Done.